modules/ml/src/test/java/org/apache/ignite/ml/math/distances/DistanceTest.java

```diff
@@ -40,6 +40,10 @@
         new EuclideanDistance(),
         new HammingDistance(),
         new ManhattanDistance(),
+        new BrayCurtisDistance(),
+        new CanberraDistance(),
+        new JensenShannonDistance(),
+        new WeightedMinkowskiDistance(4, new DenseVector(new double[]{1, 1, 1})),
```

Comment:
Maybe these parameters could become default?

Reply:
We can set default value for `p` (power) parameter as `2`, but `weight` can not be default because weight's is dimensional dependent. If we want to evaluate distance between `d(a,b)`, then `a.size() == b.size() && a.size() == weight.size()`. 
Or I can use `null` for `weight` and use them only when `weight != null`.